omod/src/main/webapp/pages/reports/activeVisitsList.gsp

```diff
@@ -26,12 +23,16 @@
     ui.includeJavascript("uicommons", "moment.min.js")
     ui.includeJavascript("pihcore", "services/configService.js")
 
-    ui.includeCss("uicommons", "angular-ui/ng-grid.min.css")
-
+    // ui.includeCss("uicommons", "angular-ui/ng-grid.min.css")
+    ui.includeCss("pihcore", "inpatient.css")
     ui.includeCss("pihcore", "activeVisitsList.css")
     ui.includeJavascript("pihcore", "visit/filters.js")
     ui.includeJavascript("pihcore", "reports/activeVisitsListController.js")
 
+    // libs for the bootstrap date picker and other libs
+//  the version in openmrs is older
```

Comment:
Oh re: my comment above, I see you've included the comment here I was talking about ... maybe just update this comment to note that we exclude the core version of bootstrap above and list the version we are using here?